.gitignore

```diff
@@ -0,0 +1,4 @@
+/.gradle/
```

Comment:
Convention for directories is `.gradle/` - note the trailing slash and no leading slashes.

Reply:
Most of the time we want to limit a scope of `.gitignore` to the particular directory level.

It doesn't seem to be an issue with `.gradle/`, but it will definitely be an issue with `build/`: consider what'll happen if we want to create a file named `src/kotlin/conan/build/file.kt` (the file will be automatically ignored).

That's why I think it's generally better to always include a leading slash when appropriate (and exclude it only in cases when we really explicitly want to remove something from the whole directory tree — although it rarely happens).